### ISSUE TITLE
No drop action available after getting stuck on environment.

### ISSUE THREAD

**Issue by WarJebus** [drakelinglabs/projectrealityarma3#379]
### ArmA3 Version:

unsure
### Mods:

also unsure
### Description:

Member of my squad was unable to drop an ammo crate he was dragging after clipping into some environment in his path.
### Steps to reproduce:

Removed ammo crate from the back of a hunter.
Dragged it away from the hunter an backed into some rubble on the ground.
Was unable to move from the rubble for a few seconds before snapping to the other side.
He could no longer drop the crate, resulting in the crate being attached to his front, but he could preform all other actions as normal.

![9pbmqto](https://cloud.githubusercontent.com/assets/1084758/19623418/af62eadc-9896-11e6-9b7e-a922217ecc49.jpg)
### Where did the issue occur?

Playtest.

**Comment by jokoho48** [comment]
Need testing can not Reproduce

**Comment by litrax** [comment]
If tested and bug can be reproduced throw it back in ToDo.

**Comment by TomNedry** [comment]
This is no issue, as the facts are false!

The screenshot provided shows not a missing drop action nor a transportation scene. While transporting cargo the players weapon is on his back, so there won't be a weapon to point at the cargo box. Also the cargo box on the provided screenshot don't hover, but stands allready on the ground.

I visited the location and took these screenshots:

- The [1st one reproduces the scene](https://www.airpressuretendency.net/pra3_qa_3jdurhtdx997sk/20161105214952_1.jpg), but as said above the crate is allready on the ground and you have to point not at the center of the crate. 
- If you [do so as seen in the 2nd screenhots](https://www.airpressuretendency.net/pra3_qa_3jdurhtdx997sk/20161105214956_1.jpg) the action menu entry can be seen. 
- In the [3rd screenshot](https://www.airpressuretendency.net/pra3_qa_3jdurhtdx997sk/20161105220344_1.jpg) one can see the scene where the crate is still transported. The only action then is to "drop", as there is no vehicle nearby and your weapon is on your back or holstered. Also no "squad screen" action...

**Comment by WarJebus** [comment]
For the sake of clarity, I had a feeling this could have been a 1 in a 10000 chance bug but although this ticket is closed, I will just add the following information if it helps later. When my squad member informed me that he couldn't drop the crate, I had him walk up to me and watched him running around with a crate floating in front of him. For all intense and purposes he was completely battle ready in the sense that he could shoot and sprint, I did not ask if he could interact with the crate although we were in fact missing a crate. As I personally wasn't the one experiencing the bug I just told him to take a screen shot with the action menu open to prove he could not drop the crate and left it at that. I apologize if I caused any problems by posting on his behalf.
